me.hanh.zwallet.Ywallet.yml

```diff
@@ -0,0 +1,48 @@
+id: me.hanh.zwallet.Ywallet
```

Comment:
`hanh.me` doesn't resolve. This doesn't meet the app-id requirements.

Reply:
Changed to `ywallet.app`